src/Writers/LaTeXWriter.jl

```diff
@@ -499,8 +499,15 @@ function _print_code_escapes_inline(io, s::AbstractString)
 end
 
 function latex(io::IO, md::Markdown.Paragraph)
-    for md in md.content
-        latexinline(io, md)
+    if occursin(r"^\h*?\\begin{\p{Xan}*?}", first(md.content))
```

Comment:
Elements of `md.content` are not guaranteed to be strings, so this may throw an error, which it shouldn't: https://travis-ci.org/github/JuliaDocs/Documenter.jl/jobs/717872752#L320-L329